server_environment_ir_config_parameter/models/ir_config_parameter.py

```diff
@@ -13,6 +13,19 @@ class IrConfigParameter(models.Model):
 
     _inherit = 'ir.config_parameter'
 
+    is_environment = fields.Boolean(
+        string="Defined by environment",
+        compute="_compute_is_environment",
+        help="If check, the value is the database will be ignored"
```

Comment:
typos:
- is the database > in the database
- the system will the use > the system will the use

Reply:
Nice improvement, thanks!

Reply:
@legalsylvain ping ;)

Reply:
done ! thanks !